app/src/main/res/values/strings.xml

```diff
@@ -1442,7 +1442,7 @@
 
     <!-- This string is displayed in the description of the Voice Search samples collect dialog. That dialog is shown the first time
          a user tries to use the voice search. -->
-    <string name="voice_samples_collect_dialog_description2">In order to improve our voice recognition and other services, we need to collect voice samples for research. We store your data securely and without identifying information. You’ll always be able to use voice search, even if you don’t allow collection.&lt;a href="privacy">Learn More&lt;/a></string>
+    <string name="voice_samples_collect_dialog_description2"><![CDATA[In order to improve our voice recognition and other services, we need to collect voice samples for research. We store your data securely and without identifying information. You’ll always be able to use voice search, even if you don’t allow collection. <a href="%1$s">Learn More</a>]]></string>
```

Comment:
`href="privacy"`

Reply:
We don't really use the variable there, I've removed it.